schema/spec_test.go

```diff
@@ -73,7 +73,7 @@ func validate(t *testing.T, name string) {
 			continue
 		}
 
-		err = schema.Validator(example.Mediatype).Validate(strings.NewReader(example.Body))
+		err = schema.Validator(example.Mediatype).Validate(strings.NewReader(example.Body), false)
```

Comment:
Yes, you are right. This test is for spec validation, which MUST be strict OCI format coincidence. `false` will lead this test case loss effectiveness. fix in https://github.com/opencontainers/image-spec/pull/452/commits/b68d69226243c79b275a3c5caeac04e3dad84231#diff-5b73117945d63fe6253dccb9c210d74fR76.